packages/back-end/src/controllers/ingestion.ts

```diff
@@ -0,0 +1,56 @@
+import { Response } from "express";
+import { SDKConnectionInterface } from "back-end/types/sdk-connection";
+import { findAllSDKConnectionsAcrossAllOrgs } from "back-end/src/models/SdkConnectionModel";
+import { AuthRequest } from "back-end/src/types/AuthRequest";
+
+interface SdkInfo {
+  organization: string;
+  client_key: string;
+  datasource: string;
+}
+
+function sdkInfo(conn: SDKConnectionInterface): SdkInfo {
+  return {
+    organization: conn.organization,
+    client_key: conn.id,
+    // TODO: pull datasource
+    datasource: "",
+  };
+}
+
+// Refresh in-mem cache every minute
+const REFRESH_INTERVAL = 60_000;
+let sdkData: Record<string, SdkInfo> = {};
+let lastUpdate = Date.now() - REFRESH_INTERVAL;
```

Comment:
There's also a cache on the ingestion endpoint which will ping this every time there's a cache miss but otherwise won't update its data